### DISCUSSION TITLE
StandAlone - Monitor Nic Selection

### DISCUSSION THREAD

**Discussion by NeoLife-iT** [Security-Onion-Solutions/securityonion#12993] [2.4] [answered]
### Version

2.4.60

### Installation Method

Network installation on Ubuntu

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

96

### Storage for /

500G

### Storage for /nsm

400G

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

I have man NiC on this Server.  I wanted to use the nic below but it keep error out - see pictures...any idea?

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36971330/08953333-8e74-4d18-a2af-101e828b6c75)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36971330/07921baf-410b-4a65-8197-af165f307de2)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Have you tried the following?

```
sudo touch /etc/NetworkManager/conf.d/10-globally-managed-devices.conf
sudo service network-manager restart
```

**Reply by NeoLife-iT** [reply]
Chris,  That's didn't work.  This is the physical Server with Real Nic that I am having this issue.

 I did this same install on hyper-V VM and it all worked fine. so the issue it is hard to narrow down.

I'm looking for a way to work around this error.

Thanks

**Reply by NeoLife-iT** [reply]
Try to set this server as Manager in new Distributed Install look like it installing.  How do I add the monitor nic later?

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36971330/c43566f3-491b-4239-b167-4da448716d86)

**Reply by NeoLife-iT** [reply]
So I got this first node installed and running.  how do I add the monitor NIC to this node?

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36971330/1d3e8d60-8175-4930-bc8d-e4ab883b9375)

**Reply by NeoLife-iT** [reply]
Trying new install as MANAGERSEARCH node 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36971330/c9141f60-05d3-4867-9f0c-190c4140b454)

**Reply by NeoLife-iT** [reply]
That install also work.  so how do i add the monitor Nic to this install?

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36971330/f5f713e6-bc91-455c-b043-712e5c69ebdf)

**Reply by NeoLife-iT** [reply]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36971330/e6573cbb-83c0-40e2-8623-fa007eaa553d)

**Reply by cm-ops** [reply]
`Manager `and `Managersearch `nodes don't monitor traffic. If you want to monitor network traffic and you are installing distributed you should choose `sensor`. https://docs.securityonion.net/en/2.4/architecture.html#distributed

**Reply by NeoLife-iT** [reply]
I did installed successfully once before on this same server.  But recently I Added more RAM and Added Intel Nic with SFP and then re-install.  It does not made sense why it is not working now.  So I am going to remove the New NIC and run-install again soon.

**Comment by dougburks** [comment]
First, please note that network installations are not officially supported:
https://docs.securityonion.net/en/2.4/network-installation.html

If the network configuration portion displays a message like ``The IP being routed by Linux is not the IP address assigned to the management interface``, then you have multiple network interfaces with IP addresses. In most cases, sniffing interfaces should not have IP addresses and there should only be an IP address on the management interface itself. Sometimes this is caused by a sniffing interface connected to a normal switch port (not a TAP/span port) and acquiring an IP address via DHCP. Double-check your network interfaces, wiring, and configuration.

**Comment by NeoLife-iT** [comment, marked as answer]
I might have mistaken on successfully install ubuntu + sOs on this physical server.  I will remove the new Nic and try the install again.
